nix/hetzner-bootstrap.nix

```diff
@@ -19,6 +19,7 @@ let
       mkdir -p "$out/bin"
       ln -s "${nix}"/bin/* "$out/bin/"
       ln -s "${stdenv.shell}" "$out/bin/sh"
+      ln -s "${pkgsNative.cacert}" "$out/etc-certs"
```

Comment:
So this is not needed since below export uses the package itself? Or is it needed elsewhere?